README.md

```diff
@@ -151,9 +151,6 @@ To enable this configuration use the extends property in your `.eslintrc` config
 {
   "extends": [
     "plugin:flowtype/recommended"
-  ],
```

Comment:
This change is made in wrong file. This file is generated using the templates in `.README` directory.